libretroshare/src/pqi/pqihandler.cc

```diff
@@ -470,18 +470,11 @@ int     pqihandler::UpdateRates()
 
 		// for our up bandwidth we take into account the max down provided by peers via BwCtrl
 		// because we don't want to clog our outqueues, the TCP buffers, and the peers inbound queues
+		mod -> pqi -> setMaxRate(false, out_max_bw);
```

Comment:
plz put a comment (basically what you said in the PR text) here. It's not really easy to figure out why you added this otherwise. thx!

Reply:
OK but we need to talk about that first.
IMO the comments already explains what we do: we limit our UP to the DN value passed by our friend via the BwCtrl service. 
And there is of course no need to mention a bug that is not there anymore.